games-strategy/0ad/0ad-0.0.22.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="O A.D. - Game"
+DESCRIPTION="A free, open-source, Real-Time Strategy (RTS) game."
+HOMEPAGE="https://play0ad.com"
+COPYRIGHT="Wildfire Games"
+LICENSE="0ADLicense"
+REVISION="1"
+SOURCE_URI="https://releases.wildfiregames.com/0ad-0.0.22-alpha-unix-build.tar.gz"
+CHECKSUM_SHA256="b50ef5aa8004bc60c542493d2c3ceff59ebdefadcd901d529c492c91503ed047"
+SOURCE_DIR="0ad-$portVersion-alpha"
+PATCHES="0ad-$portVersion-alpha.patch"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	0ad$secondaryArchSuffix = $portVersion
+	app:0ad$secondaryArchSuffix = $portVersion
+"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libnspr4$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libSDL2$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libzlib$secondaryArchSuffix
+	0ad_data == $portVersion
+"
+
+BOOST_PKG_NAME="boost${secondaryArchSuffix}_devel"
+
+if [ "$buildArchitecture" == "x86_64" ]; then
+	BOOST_PKG_NAME="boost165_devel"
+fi
+
+BUILD_REQUIRES="
+	$BOOST_PKG_NAME
+	libxml2${secondaryArchSuffix}_devel
+	nspr${secondaryArchSuffix}_devel
+	icu${secondaryArchSuffix}_devel
+"
+
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:perl
+	cmd:patch
+	cmd:python
+	cmd:cmake
+"
+
+BUILD()
+{
+	cd "build/workspaces/"
+	./update-workspaces.sh
+
+	cd gcc
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+
+	cp 0ad-$portVersion-alpha $appsDir
```

Comment:
I guess you meant this:
```
	cp 0ad-$portVersion-alpha $appsDir/0ad
```

How is the runtime supposed to find the data from 0ad_data?
Is it expecting to find a sub-directory called data in the directory it is installed?
If yes, you could try inserting this at the end of INSTALL:
```diff
+	ln -s $portPackageLinksDir/0ad_data/data/0ad $appsDir/data
 }
```

But this would not be acceptable, because $appsDir/0ad-0.0.22-alpha will appear as /system/apps/0ad-0.0.22-alpha and the symlink would be:
```
/system/apps/data -> /packages/0ad-0.0.22-1/0ad_data/data/0ad
```
In other words, we would have a "data" symlink (in /system/apps/) which would be usefull only to 0ad, but it would be visible in a system-wide directory.

The workaround would be to use this:
```
BUILD()
{
	mkdir -p $libDir/0ad
	cp 0ad-$portVersion-alpha $libDir/0ad/0ad
	ln -s $portPackageLinksDir/0ad_data/data/0ad $libDir/0ad/data
	ln -s -r $libDir/0ad/0ad $appsDir/0ad
	addAppDeskbarSymlink $libDir/0ad/0ad "0 A.D."
}
```

Some developers (but not me) would recommend this:
```
BUILD()
{
	mkdir -p $appsDir/0ad
	cp 0ad-$portVersion-alpha $appsDir/0ad/0ad
	ln -s $portPackageLinksDir/0ad_data/data/0ad $appsDir/0ad/data
	addAppDeskbarSymlink $appsDir/0ad/0ad "0 A.D."
}
```

Remark: if you rename the recipe, then you would need to replace the `0ad-$portVersion-alpha` by `0ad-${portVersion/\~/-}`